tests/test.single-discussion-with-attachment.js

```diff
@@ -31,12 +31,10 @@ fixture`Single discussion cartridge (with attachment, compact)`
   "/test-cartridges/single-discussion/imsmanifest.xml"
 )}`;
 
-test("Header and Nav is hidden", async t => {
+test("Header is hidden", async t => {
   await t
     .expect(Selector("h1").withText(`Test discussion`).exists)
     .ok()
-    .expect(Selector("nav").exists)
-    .notOk()
```

Comment:
since the purpose of your commit is to make sure the nav does not show, why remove this negative assertion?